src/scipp/experimental/plotting/wrappers.py

```diff
@@ -36,6 +36,33 @@ def plot(obj: Union[VariableLike, Dict[str, VariableLike]]) -> Figure:
         - dict of Variables
         - dict of DataArrays
         - dict of numpy ndarrays
+    aspect:
```

Comment:
Can you list all of them as actual function attributes? It would be a bit tedious but help with autocompletion, type checking, and help users when inspecting the docs or function signature. Also, you could get rid of listing the default value in the docstring.